app/actions/local/timezone.ts

```diff
@@ -23,7 +23,7 @@ export const autoUpdateTimezone = async (serverUrl: string, {deviceTimezone, use
         return {error: `No database present for ${serverUrl}`};
     }
 
-    const currentUser = await queryUserById(database, userId) ?? null;
+    const currentUser = await getUserById(database, userId) ?? null;
```

Comment:
`?? null` in not necessary at this point cause `getUserById` will return undefined